src/rubintv/handlers/external/endpoints.py

```diff
@@ -230,7 +248,13 @@ async def get_historical(request: web.Request) -> web.Response:
         smrd_dict = historical.get_events_for_date(camera, smrd)
         smrd_events = flatten_events_dict_into_list(camera, smrd_dict)
 
-        grid_columns = build_grid_columns_css(camera.channels)
+        metadata_json = "{}"
```

Comment:
This is essentially the same 7 lines of code as earlier in the code, is there a way to not repeat this?

Reply:
Thanks, all refactored now 👍